### ISSUE TITLE
Unable to set the keyboard focus to the interactive elements before to the radio button (i.e., Yes, I am willing to participate) using shift+ tab.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#288]
### Pre-requisites:
Launch the Visual Studio for Mac and Sign-in with Enterprise account.

Create projects:

1. Go to Visual Studio-> File-> New Solution. -> Multiplatform-> App-> Xamarin.Forms-> Blank Forms App.
2. Hit on Next button to create the project.
3. Hit on Debug(Play)/Build button to build/deploy the solution/project.
4. As per the choice/requirements, Change the Device platform like iOS/ Android and iPhone X iOS 11.3 etc.
5. Hit on Inspector icon button (i.e., in the top toolbar buttons of the VS) to inspect the project.
6. On hitting on inspector button, Xamarin Inspector is Launched.

### Repro-Steps:
1. Follow the Pre-requisites to Launch the Xamarin Inspector from Visual Studio.
2. Navigate to Xamarin Inspector-> preferences->Hit On Enter key, which opens the General tab by default.
3. Navigate to Feedback tab-> Hit on Enter to open Feedback tab window.
4. Navigate to "Yes , I am willing to participate (Recommended)" radio button using tab key. 
5. Observe the keyboard focus navigation, using Shift+Tab key using from Yes.. radio button.

### Expected Behavior
Keyboard focus traversing behavior should be same in the reverse and forword both the diarection.
Note: User should be able to traverse the interactive elements before to the radio button (i.e., Yes, I am willing to participate) using shift+ tab also
### Actual Behavior
While user is hitting on Shift+Tab key to traverse in the reverse direction the focus loss is observed before to the radio button (i.e., "Yes I am willing to participate" ) and keyboard focus is not observed again on any of the elements in the same diarection.
Note: The keyboard focus navigation in forword diarection is as expected and proper, so no issue is observed in forword diarection.
### User Impact
Keyboard user might be confused and face difficulty to proceed further.

![kb4_pref_feedbacktab_shifttaborderloss](https://user-images.githubusercontent.com/22956130/39983152-4c33ccb2-5774-11e8-9b03-7d524545b848.png)

### Environment
Visual Studio for Mac Enterprise [Version: 7.5 Preview (7.5 build 1222)]
Xamarin Inspector [Version: 1.4 (master/b3f92f9)]

**Comment by WiproPSAccTest** [comment]
Please add the following labels to the bug:
A11yMAS
Accessibility
Xamarin-Inspector-MAC
Wipro
MAS2.1.1

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1830
